app/assets/javascripts/views/time_table_by_day.js

```diff
@@ -0,0 +1,169 @@
+App.Views.TimeTableByDay = Backbone.View.extend({
+  el: '.time-table',
+  template: JST['templates/time_table_by_day'],
+
+  initialize: function(options) {
+    var that = this;
+
+    this.vacations = new App.Collections.TeamVacations({team_id: options.team_id});
+    this.members = new App.Collections.TeamMembers(options.team_id);
+
+    this.cellWidth  = 20;
+    this.colSpan    = 5;
+    this.timeTableDateRange = {
+      begin:  new Date(),
+      end:    new Date(),
+    };
+
+    this.updateTimeTableDateRange();
+
+    this.render();
+    this.$table = this.$('.time-table-by-day');
+
+    this.listenTo(this.members,   'sync', this.renderMembersTable);
+    this.listenTo(this.vacations, 'sync', this.renderVacationsTable);
+
+    this.members.fetch({
+      success: function() {
+        that.vacations.fetch();
+      }
+    });
+  },
+
+  render: function() {
+    this.$el.html(this.template());
+    return this;
+  },
+
+  renderMembersTable: function() {
+    var $list = this.$('.members tbody');
+    $list.append('<tr><td>&nbsp;</td></tr>');
+    $list.append('<tr><td>&nbsp;</td></tr>');
+    this.members.each(function(model) {
+      $list.append('<tr><td class="member">'+ model.composeFullName() +'</td></tr>');
+    });
+  },
+
+  renderVacationsTable: function() {
+    this.drawMonths();
+    this.drawDays();
+    this.drawEmptyTable();
+    this.markVacations();
+    return this;
+  },
+
+  drawMonths: function() {
+    var $td = null,
+        date = null,
+        cellCounter = 0,
+        monthFormat = "MMM (YYYY)",
+        cols = this.getNumberOfDays(),
+        month = moment(this.timeTableDateRange.begin).format(monthFormat);
+
+    this.$table.css('width', this.calculateTableWidth());
+
+    var $tr = $('<tr>').appendTo(this.$table);
+    $tr.addClass('head');
+
+    for (var col = 0; col < cols; col++) {
+      date = moment(this.timeTableDateRange.begin).add(col,'days');
+      if (date.format(monthFormat) !== month) {
+        $td = $('<td>').appendTo($tr);
```

Comment:
It can lead to a lot of browser's reflow. To fix it you need to create some html and make needed manipulation with that html and than insert/replace it in the DOM only once. See https://dev.opera.com/articles/efficient-javascript/?page=3#reflow